### PR TITLE
Menu - onItemRendered is not executed for sub menu items (T906117)

### PR DESCRIPTION
**Pull request by novsstation**
Bugfix T906117

### CODE REVIEW

File: js/ui/menu/ui.menu.js [hunk 1]
```diff
@@ -525,7 +526,13 @@ class Menu extends MenuBase {
             },
             onSelectionChanged: this._nestedItemOnSelectionChangedHandler.bind(this),
             onItemClick: this._nestedItemOnItemClickHandler.bind(this),
-            onItemRendered: this.option('onItemRendered'),
+            onItemRendered: (e) => {
+                if(that.option('onItemRendered')) {
```
Comment (IgnatovDan) [inline js/ui/menu/ui.menu.js]: onItemRendered: this._nestedItemOnItemRenderedHandler.bind(this), ?

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.ui.widgets/menu.tests.js [hunk 2]
```diff
@@ -2600,3 +2600,62 @@ function transferActionTest(eventName, expectedArgs, triggerFunc) {
...
+            menu.option('onItemRendered', callback);
+        } else {
+            menu.on('itemRendered', callback);
+        }
+    }
+
+    ['property', 'event'].forEach(bindingOption => {
+        QUnit.test(`itemRendered callback is called for all level nodes. Binding via ${bindingOption}`, function(assert) {
+            const expectedItemsArray = [];
+            const callback = (e) => expectedItemsArray.push(e.itemData.text);
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.ui.widgets/menu.tests.js]: https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxMenu/Configuration/#onItemRendered - check 'e.component' ?

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.ui.widgets/menu.tests.js (hunk 2 above)
Comment (novsstation) [reply testing/tests/DevExpress.ui.widgets/menu.tests.js]: Yes, fixed https://github.com/DevExpress/DevExtreme/pull/13787/commits/a2083f83e1e4988b3a450ed247b21187e9430064

File: js/ui/menu/ui.menu.js (hunk 1 above)
Comment (novsstation) [reply js/ui/menu/ui.menu.js]: Yes, fixed https://github.com/DevExpress/DevExtreme/pull/13787/commits/a2083f83e1e4988b3a450ed247b21187e9430064

**Review by IgnatovDan — DISMISSED** [review]

**Review by IgnatovDan — DISMISSED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**